test/e2e/framework/providers/gce/recreate_node.go

```diff
@@ -39,19 +39,19 @@ func nodeNames(nodes []v1.Node) []string {
 	return result
 }
 
-var _ = Describe("Recreate [Feature:Recreate]", func() {
+var _ = ginkgo.Describe("Recreate [Feature:Recreate]", func() {
 	f := framework.NewDefaultFramework("recreate")
 	var originalNodes []v1.Node
 	var originalPodNames []string
 	var ps *testutils.PodStore
 	systemNamespace := metav1.NamespaceSystem
-	BeforeEach(func() {
+	ginkgo.BeforeEach(func() {
 		framework.SkipUnlessProviderIs("gce", "gke")
 		var err error
 		numNodes, err := framework.NumberOfRegisteredNodes(f.ClientSet)
-		Expect(err).NotTo(HaveOccurred())
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
```

Comment:
Oops, sorry for missing this in my previous comment.
Can we replace here with just `framework has ExpectNoError(err)` ?

Reply:
done.